styles.css

```diff
@@ -0,0 +1,209 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: white;
+}
+
+.header-container {
+  background-image: url("images/header-image.jpg");
+  background-repeat: no-repeat;
+  display: flex;
+  background-color: grey;
+  width: 100%;
+  height: 30rem;
+  position: relative;
+}
+
+.header-logo {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  margin-left: 50px;
+  width: 20%;
+
+}
+
+.header-container p {
+  color: #81B741;
+  font-size: 1.2rem;
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.cart-container {
+  display: flex;
+
+}
+
+
+.cart-icon {
+  display: flex;
+  width: 2rem;
+  position: absolute;
```

Comment:
the positioning in your header might be a little off. Id recommend taking it out and maybe use padding, margin and flexbox to position things. Position can be a little tricky.